From d3e59724601b2aabb231f2e12e16ebf00ba79ab1 Mon Sep 17 00:00:00 2001
From: Caleb Marble <cm@marblenix.com>
Date: Sun, 2 Mar 2014 13:53:05 -0600
Subject: [PATCH] modified:   ext/ncurses/ncurses_wrap.h
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fixes the following compiler error:

ext/ncurses/ncurses_wrap.c: In function ‘rbncurshelper_nonblocking_wgetch’:
ext/ncurses/ncurses_wrap.c:807:12: error: variable ‘tz’ has initializer but incomplete type
     struct timezone tz = {0,0};

Also closes #6
---
 ncurses_wrap.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ncurses_wrap.h b/ncurses_wrap.h
index 85da4f8..92583c0 100644
--- a/ncurses_wrap.h
+++ b/ncurses_wrap.h
@@ -84,6 +84,7 @@ int close(int);
 #endif
 
 #include <ruby.h>
+#include <sys/time.h>
 
 extern VALUE mNcurses;  /* module Ncurses */
 extern VALUE cWINDOW;   /* class Ncurses::WINDOW */
-- 
1.9.1

